source/agora/consensus/state/ValidatorSet.d

```diff
@@ -163,27 +152,6 @@ public class ValidatorSet
         return null;
     }
 
-    /***************************************************************************
-
-        Returns:
-            the number of active enrollments, or 0 if there was a database error
-
-    ***************************************************************************/
-
-    public size_t count () @trusted nothrow
-    {
-        try
-        {
-            return this.db.execute("SELECT count(*) FROM validator_set WHERE active = ?", EnrollmentStatus.Active).
-                oneValue!size_t;
-        }
-        catch (Exception ex)
-        {
-            log.error("Error while calling ValidatorSet.count(): {}", ex);
-            return 0;
-        }
-    }
-
```

Comment:
Unused I guess ? It's own commit then.

Reply:
It was used but had a duplicate. I will extract into separate commit.